File: FrontEnd/src/app/models/document.dto.ts
import { UserDTO } from './user.dto';
export interface DocumentDTO {
  id: number;
  type: string;
  createdAt: string;
  updatedAt: string;
  status: string;
  user: UserDTO;
  comment?: string;
}
export interface DocumentUpdateDTO {
  status: string;
  comment?: string;
  documentId: number;
}

File: BackEnd/src/main/java/com/example/futurbe/dto/Documents/DocumentDTO.java
package com.example.futurbe.dto.Documents;

import com.example.futurbe.entitys.DocumentStatus;
import com.example.futurbe.entitys.DocumentType;
import com.example.futurbe.entitys.User;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DocumentDTO {
    private Long id;
    private DocumentType type;
    private Date updatedAt;
    private DocumentStatus status;
//    private User user;
    private Date createdAt;
    private String comment;

}
